play-ws-standalone/src/main/java/play/libs/ws/WSResponseHeaders.java

```diff
@@ -3,54 +3,7 @@
  */
 package play.libs.ws;
 
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-
 public interface WSResponseHeaders {
```

Comment:
I think this should be removed completly since it's no longer in use (or deprecated)